src/views/Home/components/Milestones/index.view.tsx

```diff
@@ -1,8 +1,39 @@
 import React from "react";
 import "./milestones.scss";
 
+const milestones = [
+  {
+    title: "Attendees",
+    text: "500+",
+  },
+  {
+    title: "First-time Hackers",
+    text: "47%",
+  },
+  {
+    title: "Projects",
+    text: "60+",
+  },
+  {
+    title: "In Prizes",
+    text: "$20,000+",
+  },
+];
+
 const MilestonesComponent: React.FC = () => {
-  return <div className="Milestones">Hello from MilestonesComponent</div>;
+  return (
+    <div className="Milestones">
+      <h1 className="Milestones__title">Milestones of 2020</h1>
+      <div className="Milestones__container">
+        {Object.entries(milestones).map(([_, milestone]) => (
+          <div className="Milestones__box" key={_}>
+            <h1 className="Milestones__box--top">{milestone.text}</h1>
+            <p className="Milestones__box--bottom">{milestone.title}</p>
+          </div>
```

Comment:
Should we change `h1` and `p` to `div` to be more consistent throughout the application? 

Out naming convention also associates `--` with variations - is this the correct way to use it here?